app/views/publishers/vacancies/job_applications/reference_requests/_reference_not_provided.html.slim

```diff
@@ -0,0 +1 @@
+= govuk_warning_text(text: t(".this_referee_will_not_provide_a_reference"))
```

Comment:
(Not a request, just a thought) :thinking:  Is it worth having a dedicated partial for this one-line warning?